examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/template-types/af-page-v2/policies/.content.xml

```diff
@@ -5,18 +5,22 @@
             cq:policy="forms-components-examples/components/page/default"
             jcr:primaryType="nt:unstructured"
             sling:resourceType="wcm/core/components/policies/mappings">
-        <parsys1
-            cq:policy="wcm/foundation/components/responsivegrid/page-header"
-            jcr:primaryType="nt:unstructured"
-            sling:resourceType="wcm/core/components/policies/mapping"/>
+        <container1
+                jcr:primaryType="nt:unstructured"
+                jcr:title="Header"
+                sling:resourceType="wcm/core/components/policies/mapping"
+                cq:policy="wcm/foundation/components/responsivegrid/page-header"
+                layout="responsiveGrid"/>
```

Comment:
How changing name from parsys1 to container1 makes difference?

Reply:
It will not make any difference here but it may create confusion because we will not use parsys anywhere else